config/settings/base.py

```diff
@@ -470,6 +470,11 @@
 
 LOOKASIDE_CACHE_BASE_URL = os.getenv("CORGI_LOOKASIDE_CACHE_URL")
 
+# Pyxis details
+PYXIS_GRAPHQL_URL = os.getenv("CORGI_PYXIS_GRAPHQL_URL")
+PYXIS_CERT = os.getenv("CORGI_PYXIS_CERT")
```

Comment:
`os.getenv` has a mypy type of `Optional[str]` because it can return `None` if there's no matching variable, I think that's probably the cause of the error you encountered. If you add a default value like `""`, mypy should be happy.

Reply:
Good find! Fixed in 8bc8a96.